### ISSUE TITLE
Extentions of API to add datamodel with different types

### ISSUE THREAD

**Issue by IPrototypeI** [splunk/terraform-provider-splunk#147]
Would be possible to add the feature to add datamodels with different types via terraform?

Or can you maybe tell me which API I have to use so I can implement it myself and provide a pull request ?

Because services/datamodel/model does not include the display mode

**Comment by anushjay** [comment]
Yes, this would be a great addition but we are currently not actively working on adding features. Happy to take contributions if you are able to add support.
Did you take a look at the knowledge endpoints? There are some data model related API [available](https://docs.splunk.com/Documentation/Splunk/9.0.0/RESTREF/RESTknowledge).

**Comment by IPrototypeI** [comment]
@anushjay yes I did and I was already able to implement this feature I would be open to contribute to the plugin.
I can create a pullrequest, btw would you be open to discuss some feature implementation with me, too ?

Because sometimes I am not sure if we have any workarounds for reducing the effort with calling the API, because the api is not documented well enough and I have to extract the calls via the network trace on the splunk ui to get more informations on the payload
